string id
int32 timestamp
string meta_data
string waypoint

#sensor_msgs/PointCloud2 metaroom_filtered_cloud
octomap_msgs/Octomap metaroom_filtered_octomap
sensor_msgs/PointCloud2 up_facing_points
#sensor_msgs/PointCloud2[] sweep_clouds
sensor_msgs/Image[] sweep_imgs
octomap_msgs/Octomap segmented_objects_octomap
